lua/tabnine/binary.lua

```diff
@@ -41,11 +41,10 @@ local function binary_path()
 	end, fn.glob(binaries_path .. "/*", true, true))
 
 	paths = vim.tbl_map(function(path)
-		return semver(path)
+		return semver.parse(path)
 	end, paths)
 
 	table.sort(paths)
```

Comment:
Does `table.sort()` work on these objects? have we checked this?

Reply:
Nevermind, having tested myself, it seems that it does, in fact, work. I wonder if internally it's using the __lt/__gt methods...

Reply:
Oh yes.. it does :)